docs/API.md

```diff
@@ -820,6 +822,51 @@ Example: `GET wordpress/meta/charm-metadata`
 }
 ```
 
+#### GET *id*/meta/charm-metrics
+
+The `/meta/charm-metrics` path returns the contents of the charm metrics file
+for a charm, or an empty response if no metrics are defined for the charm. The
```

Comment:
That's not quite true. As specified in the "Bulk requests and missing metadata" section, you'll get a 404 (missing metadata) response when the singular form of the endpoint is used, or an omitted entry otherwise.